src/pages/LinkedAccounts/LinkedAccounts.tsx

```diff
@@ -0,0 +1,247 @@
+  }, [activeWorkspace]);
+
+  const getAccounts = async () => {
+    const res = await fetch(`/api/workspaces/${activeWorkspace.name}/linked-accounts`);
+    if (!res.ok) {
+      setError('Failed to get linked accounts');
+      return;
+    }
+    const accounts: string[] = await res.json();
+    const initial: AccountMetaData[] = [];
+    linkableAccounts.forEach((account) => {
+      initial.push({
+        key: accounts.includes(account.internalName) ? 'the_user_will_never_see_this' : '',
+        linked: accounts.includes(account.internalName) ? true : false,
+        message: accounts.includes(account.internalName) ? 'Linked' : 'Not linked',
+        internalName: account.internalName,
+        externalName: account.externalName,
+      });
+    });
+    setData(initial);
+  };
+
+  // Internal Dev only
+  const getPlaceHolderAccounts = () => {
+    const initial: AccountMetaData[] = [
+      {
+        key: 'the_user_will_never_see_this',
+        linked: true,
+        message: 'Linked',
+        internalName: 'airbus',
+        externalName: 'Airbus',
+      },
+      {
+        key: '',
+        linked: false,
+        message: 'Not linked',
+        internalName: 'planet',
+        externalName: 'Planet',
+      },
+    ];
+    setData(initial);
+  };
+
+  const renderHeader = () => {
+    return (
+      <div className="header">
+        <div className="header-left">
+          <h2>Linked accounts</h2>
+        </div>
+        <div className="header-right">
+          <img alt="Members" src={link} />
+          <div className="header-right-text">
+            <span className="header-right-title">Linked accounts</span> allows you to store your
+            provider API key in the Hub allowing you to order commercial data.
+          </div>
+        </div>
+      </div>
+    );
+  };
+
+  const updateData = (account: AccountMetaData, key: string, value: string | boolean) => {
+    const copy = [...data];
+    const index = copy.findIndex((a) => a.internalName === account.internalName);
+    copy[index][key] = value;
+    setData(copy);
+  };
+
+  const renderAccount = (account: AccountMetaData) => {
+    return (
+      <div className="linked-accounts__account">
+        <div className="linked-accounts__account-header">
+          {account.externalName} <span>| {account.message}</span>
+        </div>
+        <div className="linked-accounts__account-input">
+          <input
+            disabled={account.linked}
+            placeholder="Enter your contact / api key"
+            type={account.linked ? 'password' : 'text'}
+            value={account.key}
+            onChange={(e) => {
+              updateData(account, 'key', e.target.value);
+            }}
+          />
+        </div>
+        {account.linked ? renderUnlinkButton(account) : renderLinkButton(account)}
+      </div>
+    );
+  };
+
+  const renderLinkButton = (account: AccountMetaData) => {
+    return (
+      <Button
```

Comment:
I'm unsure why but the button appears really small on my screen, we will need to make it bigger